src/main/java/pl/danyboy/GuessStage.java

```diff
@@ -2,25 +2,23 @@
 
 public class GuessStage {
 
-    private final LineReader lineReader;
     private final PasswordManager passwordManager;
 
-    public GuessStage(LineReader lineReader, PasswordManager passwordManager) {
-        this.lineReader = lineReader;
+    public GuessStage(PasswordManager passwordManager) {
         this.passwordManager = passwordManager;
     }
 
-    public void passwordGuessesInformation(String password) {
-        String charOrPasswordFromUser = transformToLower(lineReader.readLineFromUser());
+    public void passwordGuessesInformation(String password, String inputFromUser) {
+
+        String charOrPasswordFromUser = transformToLower(inputFromUser);
         if (charOrPasswordFromUser.length() <= 1) {
             System.out.println("Zgaduję literę");
             System.out.println(transformToLower(password).contains(charOrPasswordFromUser) ? "Zgadnięta" : "Taka litera nie występuje w haśle");
```

Comment:
```
System.out.println(transformToLower(password).contains(charOrPasswordFromUser) ? "Zgadnięta" : "Taka litera nie występuje w haśle");
```
Ta linijka to trochę przerost formy nad treścią moim zdaniem. Zdecydowanie czytelniejszy będzie zwyczajny if(..) { "zgadnieta" } else {..}